application/core/LSUploadHelper.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/*
+ * LimeSurvey
+ * Copyright (C) 2020 The LimeSurvey Project Team / Carsten Schmitz
+ * All rights reserved.
+ * License: GNU/GPL License v2 or later, see LICENSE.php
+ * LimeSurvey is free software. This version may have been modified pursuant
+ * to the GNU General Public License, and as distributed it includes or
+ * is derivative of works licensed under the GNU General Public License or
+ * other free or open source software licenses.
+ * See COPYRIGHT.php for copyright notices and details.
+ * 
+ */
+
+class LSUploadHelper
```

Comment:
Can we move this to service folder instead? The service folder has additional checks for code quality, would be nice if new code could use those checks.

Reply:
Also, no need to prefix with "LS" since we have a LimeSurvey namespace in the service folder.

Reply:
What about now?

Reply:
Travis failed. @gabrieljenik can you check, pelase?

Reply:
I added some space fixes. I think it was that. Let's see...

Reply:
You can run all Travis syntax checks locally, with `composer install; composer test`.

Reply:
Latest error is namespace error in docblock: `Exception` vs `\Exception`, _or_ `use Exception;` at top of file.